examples/BSTLD.py

```diff
@@ -39,7 +39,7 @@
 """"""
 
 """Upload Dataset"""
-dataset_client = gas.upload_dataset(dataset)
+dataset_client = gas.upload_dataset(dataset, jobs=8, skip_uploaded_files=False)
```

Comment:
I think there is no necessary to add `skip_uploaded_files` to example code~
I think `skip_uploaded_files` is quite a big feature, just mention it in the `Upload Data` chapter.
@Hoteryoung

Reply:
Indeed a better way.
@QianBao8902 Please delete the skip_uploaded_files argument in the files you changed in this commit.